circle.yml

```diff
@@ -10,10 +10,13 @@ dependencies:
   post:
     - mv ./gopherjs $HOME/bin
     - npm install --global node-gyp
+    - npm install
```

Comment:
I'm not familiar with npm, but why `npm install` is required here? (comments would be helpful)